specification/attributes/null_handling.md

```diff
@@ -19,7 +19,7 @@ Indicates how to handle columns that don't have a value.
 ## Requirements
 
 * Columns MUST use NULL when there isn't a value that can be specified for a nullable column.
-* Columns MUST NOT use empty strings or placeholder values such as 0 for numeric columns or "Not Applicable" for string columns, regardless of whether the column allows nulls or not.
+* Columns MUST NOT use empty strings or placeholder values to represent a (NULL/No Value) such as 0 for numeric columns or "Not Applicable" for string columns, regardless of whether the column allows nulls or not.
```

Comment:
The casing and parentheses here seem odd. Would this convey the same intent...? Other than that, this conceptually looks good.
```suggestion
* Columns MUST NOT use empty strings or placeholder values to represent not having a value such as 0 for numeric columns or "Not Applicable" for string columns, regardless of whether the column allows nulls or not.
```